src/Http/Http.Abstractions/src/RouteHandlerInvocationContext.cs

```diff
@@ -7,14 +7,14 @@ namespace Microsoft.AspNetCore.Http;
 /// Provides an abstraction for wrapping the <see cref="HttpContext"/> and parameters
 /// provided to a route handler.
 /// </summary>
-public class RouteHandlerFilterContext
+public class RouteHandlerInvocationContext
```

Comment:
```suggestion
public sealed class RouteHandlerInvocationContext
```

Reply:
We don't want to seal this. We want to do the crazy generic trick